server/index.js

```diff
@@ -2,13 +2,38 @@ require('dotenv/config');
 const express = require('express');
 const staticMiddleware = require('./static-middleware');
 const errorMiddleware = require('./error-middleware');
-
+const jsonMiddleware = express.json();
+// const ClientError = require('./client-error');
+const pg = require('pg');
 const app = express();
 
 app.use(staticMiddleware);
+app.use(jsonMiddleware);
+
+const db = new pg.Pool({ // eslint-disable-line
```

Comment:
What lint warning is this suppressing? It's good to have VS Code generate the disable (click on the lightbulb) so that it indicates exactly which warning is being suppressed.